.github/workflows/rspec.yml

```diff
@@ -0,0 +1,34 @@
+name: rspec tests
+
+on: [pull_request]
+
+jobs:
+  rspec:
+    name: Rspec
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports: ["5432:5432"]
+        env:
+          POSTGRES_DB: pangolin_test
+          POSTGRES_USER: pangolin
+          POSTGRES_PASSWORD: postgres
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: ./.github/actions/setup-project
+
+      - id: database-setup
+        uses: ./.github/actions/setup-database
+
+      - name: Run rspec
+        env:
+          DATABASE_URL: ${{ steps.database-setup.outputs.database_url }}
```

Comment:
[praise] I wish this were a _little_ more succinct, but I think you've made it as short and easy as possible. Love it.